Documentation/hacking.md

```diff
@@ -210,7 +247,7 @@ Finally, here we are, the magic command, the holy grail, the ultimate conclusion
 Provided you have followed the preceding instructions, regardless of whether you are adding/removing/modifying dependencies, this command will cast the necessary spells to solve all of your dependency worries:
```

Comment:
You can update this silly stanza now (or put it into the script itself above the save command..)